README.md

```diff
@@ -99,7 +123,12 @@ $("button").on("click", function(event){
 
 Your Answer:
 ```js
-// write code here
+
+document.getElementById("button").addEventListener("click", function(){
+   document.getElementById('button').style.color = "red";
+});
+
+
 ```
 
```

Comment:
unfortunately, in vanilla js you can't add an event listener to a collection, so you need to loop through all of the elements and add an event listener to each individual element